packages/cross_file/run_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This script runs browser tests for this package,
+# as described on the README.md.
+# This script is run by the `custom-test` CI step.
+# VM unit tests are run by the `test` check.
+
+set -e
+dart test -p chrome
```

Comment:
The goal is to eliminate `run_test.sh` support, and have everything use `run_test.dart`. Was there a reason it couldn't be in Dart, or was it just that this was the easiest thing to do?

It's fine to do this for now if it's the latter, since it's not like a one-line shell script is real tech debt, I'm just wondering if there was an issue I should be aware of for eventual migration.

Reply:
Nope, I discovered run_test.dart too late (actually in an error in the Windows execution saying that only .dart was supported there). I'll make this a run_test.dart. I wasn't sure on the location of the script, though, `pigeon` seems to have both :P

Reply:
`/run_test.sh` is a legacy thing that was just hand-rolled at some point in this repo (with a one-off repo-level shell script driving it), and then other packages started using. We started converting to Dart in Pigeon, both to better support running on Windows and because non-trivial shell scripts (like Pigeon's is) are a nightmare to maintain, and I added a repo tool command that drives it instead, and had it yell about the shell version being deprecated.

Pigeon is half-migrated, with some calling between them. It's slow because it's a big script :/

I should probably do it as an allow-list instead of just a log message so it's impossible to add new ones.

Reply:
I've replaced this with a dart script; I haven't used dart:io much, so I'll ask for another review of this (hoping that there's a less verbose way of doing what I want to do :P)